recipes/image-meta-tag/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "image-meta-tag" %}
+{% set version = "0.6.3" %}
+{% set giturl = "https://github.com/SciTools-incubator/image-meta-tag" %}
+{% set sha256 = "8670b421aa55102b4e7de6b845bc655f9d35e8659c7fd2a4f326fa4347b21f99" %}
+
+package:
+  name: image-meta-tag
+  version:  {{ version }}
+
+source:
+  git_url: {{ giturl }}
+  git_tag: {{ version }}
+  fn: {{ version }}.tar.gz
+  url: {{ giturl }}/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+      - python >=2.7,<3
```

Comment:
Just use `python` here without versions and then put:

```
skip: True  # [py3k]
```

in the `build` section above.